typescript/hardhat/src/TestAbacusDeploy.ts

```diff
@@ -50,14 +56,35 @@ export class TestAbacusDeploy extends TestDeploy<
   async deployInstance(domain: types.Domain): Promise<TestAbacusInstance> {
     const signer = this.config.signer[domain];
     const signerAddress = await signer.getAddress();
-    const validatorManagerFactory = new ValidatorManager__factory(signer);
-    const validatorManager = await validatorManagerFactory.deploy();
-    await validatorManager.enrollValidator(domain, signerAddress);
-    await Promise.all(
-      this.remotes(domain).map(async (remote) =>
-        validatorManager.enrollValidator(remote, signerAddress)
-      )
-    );
+
+    const outboxMultisigValidatorManagerFactory =
+      new OutboxMultisigValidatorManager__factory(signer);
+    const outboxMultisigValidatorManager =
+      await outboxMultisigValidatorManagerFactory.deploy(
+        domain,
+        [signerAddress],
+        1
+      );
+
+    const inboxMultisigValidatorManagerFactory =
+      new InboxMultisigValidatorManager__factory(signer);
+    const inboxMultisigValidatorManagers: Record<
+      types.Domain,
+      InboxMultisigValidatorManager
+    > = {};
+
+    // this.remotes reads this.instances which has not yet been set.
+    const remotes = Object.keys(this.config.signer).map((d) => parseInt(d));
+    const inboxMultisigValidatorManagerDeploys = remotes.map(async (remote) => {
```

Comment:
This is a lot of additional contracts to deploy, and if `abacus.deploy()` is put in a beforeEach hook, could slow down tests substantially.

Do we need to deploy *any* validator managers for the hardhat plugin? Arguably they're not useful since we don't require checkpoints in order to process messages on TestInbox.

Reply:
Good point, just took it out